src/main/java/org/sagebionetworks/bridge/spring/controllers/CRCController.java

```diff
@@ -189,47 +208,48 @@ public StatusMessage updateParticipant(@PathVariable String userId) {
-        httpBasicAuthentication();
+        App app = httpBasicAuthentication();
         
         IParser parser = FHIR_CONTEXT.newJsonParser();
         JsonNode data = parseJson(JsonNode.class);
         Observation observation = parser.parseResource(Observation.class, data.toString());
         
         String userId = findUserId(observation.getIdentifier());
         
-        int status = writeReportAndUpdateState(userId, data, OBSERVATION_REPORT, AccountStates.TESTS_AVAILABLE);
+        int status = writeReportAndUpdateState(app, userId, data, OBSERVATION_REPORT, AccountStates.TESTS_AVAILABLE);
         if (status == 200) {
             return ResponseEntity.ok(new StatusMessage("Observation updated."));
         }
         return ResponseEntity.created(URI.create("/v1/cuimc/observations/" + userId))
                 .body(new StatusMessage("Observation created."));        
     }
     
-    void createLabOrder(StudyParticipant participant) {
+    void createLabOrder(Account account) {
         // Call external partner here and submit the patient record
         // this will trigger workflow at Columbia.
-        /*
-        Patient patient = createPatient(participant);
+        Patient patient = createPatient(account);
         IParser parser = FHIR_CONTEXT.newJsonParser();
         String json = parser.encodeResourceToString(patient);
         
         try {
-            HttpResponse response = Request.Post("<url unknown>")
-                    .bodyString(json, APPLICATION_JSON)
-                    .execute()
-                    .returnResponse();
+            HttpResponse response = post(json);
             if (response.getStatusLine().getStatusCode() != 200 && 
                 response.getStatusLine().getStatusCode() != 201) {
-                LOG.error("Error submitting patient record to CUIMC for user " + participant.getId());
-                throw new BridgeServiceException("The server encountered an error");
+                LOG.error(EXT_ERROR + account.getId());
+                throw new ServiceUnavailableException(UNAVAILABLE_ERROR);
             }
-            // Anything to persist here?
-            LOG.info("Patient record submitted to CUIMC for user " + participant.getId());
+            LOG.info("Patient record submitted to CUIMC for user " + account.getId());
         } catch (IOException e) {
-            LOG.error("Error submitting patient record to CUIMC for user " + participant.getId());
-            throw new BridgeServiceException("The server encountered an error");
+            LOG.error(EXT_ERROR + account.getId());
+            throw new ServiceUnavailableException(UNAVAILABLE_ERROR);
         }
-        */
+    }
+    
+    HttpResponse post(String bodyJson) throws IOException {
+        return Request.Post("<url unknown>")
+                .bodyString(bodyJson, APPLICATION_JSON)
+                .execute()
+                .returnResponse();
```

Comment:
Consider disabling this altogether until we have a real URL. This way, we won't get spurious errors when someone calls this API and forget to tag the participant with test_user.